build.gradle

```diff
@@ -2,8 +2,8 @@ buildscript {
     ext {
         pluginVersion = rootProject.properties['pluginVersion']
         esVersion = rootProject.properties['esVersion']
-        keycloakVersion = '4.5.0.Final'
-        jacksonVersion = '2.8.10'
+        keycloakVersion = '9.0.2'
```

Comment:
There is a PR up for updating our project to 11.0.0

Reply:
Okay, we can update this to 11 once that is in.  I do not expect it to effect the plugin.

Reply:
You can ignore that for now. I declined it since I was just playing around with it.